worlds/ahit/__init__.py

```diff
@@ -119,7 +129,10 @@ def create_regions(self):
         # place vanilla contract locations if contract shuffle is off
         if not self.options.ShuffleActContracts:
             for name in contract_locations.keys():
-                self.multiworld.get_location(name, self.player).place_locked_item(create_item(self, name))
+                loc = self.multiworld.get_location(name, self.player)
```

Comment:
not necessary but available if you want to use it
```suggestion
                loc = self.get_location(name)
```